code/modules/uplink/uplink_items.dm

```diff
@@ -886,16 +886,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "Stimpacks, the tool of many great heroes, make you nearly immune to stuns and knockdowns for about \
 			5 minutes after injection."
 	item = /obj/item/reagent_containers/syringe/stimulants
-	cost = 5
+	cost = 3
```

Comment:
Y tho

Reply:
One use, and rather dull for affects

Reply:
They shouldnt be costing more then adrenals and be shit at the same time. Really this is only good if you get it in a bundle/space ruin